docs: add Azure Firewall scrape (#64)
promitor/docs: docs/scraping/providers/azure-firewall.md

---
tags:
  - Scraper
  - Resource Discovery
  - Azure Firewall
---

> # Azure Automation account

Reviewer: Can you align please?

Author: Hi, fixed!

> ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.1-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)

Reviewer: ```suggestion
![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.12-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```

> You can scrape an Azure Automation account via the `AzureFirewall`

Reviewer: ```suggestion
You can scrape an Azure Firewall via the `AzureFirewall`
```

 resource type.

When using declared resources, the following fields need to be provided:

- `azureFirewallName` - The name of the Azure Firewall.

All supported metrics are documented in the official [Azure Monitor documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-network-azurefirewalls-metrics).

> 
> ## Example

Reviewer: Nit
```suggestion
## Example
```

Here is an example configuration:

> 
> 

Reviewer: Nit
```suggestion

```
